src/pages/TimetablePage/components/TimetableList/TimetableSettingModal/index.tsx

```diff
@@ -25,7 +25,7 @@ export default function TimetableSettingModal({
 }: TimetableSettingModalProps) {
   const token = useTokenState();
   const semester = useSemester();
-  const toast = useToast();
+  const myLectures = useMyLecturesV2(focusFrame.id!);
```

Comment:
이렇게 쓰면 id가 undefined일 때 런타임에러가 발생할 것 같아요. 이렇게 작성하는건 어떤가요?
```suggestion
  const myLectures = focusFrame.id ? useMyLecturesV2(focusFrame.id) : null;

```

Reply:
```
React Hook "useMyLecturesV2" is called conditionally. React Hooks must be called in the exact same order in every 
 component render.
```
라는 lint에러가 나네요! `TimetableSettingModal`이 시간표 프레임의 설정 버튼을 눌러 생기는 컴포넌트라 `focusFrame`이 무조건 있다는 것을 보장할 수 있습니다! 이러한 경우에도 '!' 사용을 자제하는 것이 좋을까요?

Reply:
lint에서 막는군요..
해당 값이 확실히 보장된다면 `!`를 사용해도 에러가 발생하진 않을 것 같긴해요